pkg/kopia/command/policy_show_global.go

```diff
@@ -16,7 +16,7 @@ package command
 
 type PolicyShowGlobalCommandArgs struct {
 	*CommandArgs
-	GetJsonOutput bool
+	GetJsonOutput bool //nolint:stylecheck // Validate workflows before enabling stylecheck
```

Comment:
@ankitjain235 Is it a todo comment? The explanation is not clear

Reply:
refactored